docker-compose.observability.yml

```
@@ -0,0 +1,55 @@
+    networks:
+      - microservices-network
+    healthcheck:
+      test: ["CMD", "wget", "--no-verbose", "--tries=1", "--spider", "http://localhost:16686/"]
+      interval: 30s
+      timeout: 5s
+      retries: 3
+
+networks:
+  microservices-network:
+    external: true
+    name: high_availability_architecture_microservices-network 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)